docs/pages/management/security.mdx

```diff
@@ -1,11 +1,24 @@
 ---
-title: Security
-description: Hardening the security of your Teleport deployment
-layout: tocless-doc
+title: Secure Practices for Teleport Clusters
+description: Highlights recommended practices and ways to harden security for your Teleport cluster.
 ---
 
-- [Reducing the blast radius of attacks](./security/reduce-blast-radius.mdx).
+To help ensure that Teleport can protect your infrastructure in a production environment, you should 
+be aware of and follow recommended security practices and address potential security issues to reduce 
+system vulnerabilities and to avoid security incidents. In most cases, there are tradeoffs between 
+convenience and security that you should take into consideration.
+
+The topics in this section are intended to help secure your Teleport cluster in a way that's acceptable and 
+reasonable for your organization.
+
+You should note that the security practices covered in this section aren't necessarily reflected in the 
+examples used in the documentation. Examples in the documentation are primarily intended for demonstration 
+purposes and for development environments.
+
+- [Privilege Elevation](./security/privilege-elevation.mdx). Learn about potential risks of allowing privileged 
```

Comment:
`Privilege Elevation` sounds like you are elevating your privileges beyond what was assigned.  I think `Elevated Privileges` or `Sensitive Access` or `Access Risks` sound more accurate.